src/odemis/driver/test/tescan_test.py

```diff
@@ -133,6 +145,177 @@ def test_pickle(self):
+        f.result()
+        testing.assert_pos_not_almost_equal(self.stage.position.value, start_pos, atol=0.1e-6)
+
+        new_pos = self.stage.position.value.copy()
+        f = self.stage.moveRel({"z": 1e-6, "rx": math.radians(0.1), "rz": math.radians(0.1)})
+        f.result()
+        testing.assert_pos_not_almost_equal(self.stage.position.value, new_pos, atol=0.1e-6)
+
+        new_pos = self.stage.position.value.copy()
+        f = self.stage.moveRel({"z": 1e-6, "rx": math.radians(0.1), "rz": math.radians(0.1)})
+        f.result()
+        testing.assert_pos_not_almost_equal(self.stage.position.value, new_pos, atol=0.1e-6)
+
+        # go back the same amount of movements and axes values
+        new_pos = self.stage.position.value.copy()
+        f = self.stage.moveRel({"z": -1e-6, "rx": -math.radians(0.1), "rz": -math.radians(0.1)})
+        f.result()
+        testing.assert_pos_not_almost_equal(self.stage.position.value, new_pos, atol=0.1e-6)
+
+        new_pos = self.stage.position.value.copy()
+        f = self.stage.moveRel({"z": -1e-6, "rx": -math.radians(0.1), "rz": -math.radians(0.1)})
+        f.result()
+        testing.assert_pos_not_almost_equal(self.stage.position.value, new_pos, atol=0.1e-6)
+
+        new_pos = self.stage.position.value.copy()
+        f = self.stage.moveRel({"z": -1e-6, "rx": -math.radians(0.1), "rz": -math.radians(0.1)})
+        f.result()
+        testing.assert_pos_not_almost_equal(self.stage.position.value, new_pos, atol=0.1e-6)
+        # test if after the last movement, the position is the same as the starting position
+        testing.assert_pos_almost_equal(self.stage.position.value, start_pos, atol=0.1e-6)
+
+    def test_overflow_and_underflow(self):
+        """
+        With a relative move starting from the absolute zero point move a bit under the range
+        and afterward move a bit over the range. This is tested for the full rotational axis Rz only.
+        """
+        # start with going to the absolute zero position
+        self.stage.moveAbsSync({"rz": 0.0})
+        start_pos = self.stage.position.value.copy()
+
+        # test if underflow of the range is picked up
+        f = self.stage.moveRel({"rz": -math.radians(3)})
+        f.result()
+        self.assertLess(start_pos["rz"], self.stage.position.value["rz"])
+
+        # test if overflow of the range is picked up
+        f = self.stage.moveRel({"rz": math.radians(6)})
+        f.result()
+        self.assertGreater(self.stage.position.value["rz"], start_pos["rz"])
+
+    def test_stop_big_move(self):
+        """
+        Test if it's possible to stop the stage while it's moving by using a big move.
+        """
+        start_pos = self.stage.position.value.copy()
+
+        # check a big move and stop it
+        f = self.stage.moveRel({"rz": math.radians(180)})
+        f.result()
+
+        # wait for 1 second and stop the movement
+        time.sleep(0.2)
```

Comment:
In the comment it says "wait for 1 second" but the sleep duration is 0.2 seconds. The mismatch needs to be fixed.